src/features/payment/components/IframeCardForm/IframeCardForm.tsx

```diff
@@ -0,0 +1,286 @@
+import React from "react";
+import { Box } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import {
+  getSessionItem,
+  SessionItems,
+} from "../../../../utils/storage/sessionStorage";
+import { FormButtons } from "../../../../components/FormButtons/FormButtons";
+import { PaymentMethod } from "../../../../features/payment/models/paymentModel";
+import { Field, RenderField } from "./IframeCardField";
+
+interface Props {
+  loading?: boolean;
+  onCancel: () => void;
+  onSubmit?: (bin: string) => void;
+  hideCancel?: boolean;
+}
+
+interface FieldFormStatus {
+  isValid: boolean;
+  errorCode: null | string;
+  errorMessage: null | string;
+}
+
+interface BuildResp {
```

Comment:
missing `paymentMethod` according to openapi https://github.com/pagopa/pagopa-infra/pull/1241

Reply:
https://editor.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pagopa-infra/3c86e4514a4f06fad2668b764ee0c4699b7195fd/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl